packages/shared/src/dto/stateless-control-values/stateless-workflow-to-step-control-values.ts

```diff
@@ -0,0 +1,10 @@
+export class StatelessWorkflowToStepControlValues {
```

Comment:
How about making it shorter?
```suggestion
export class StatelessControls {
```

We have to stick to a single name for control, control values, and control schemas in the codebase. My suggestion is that these are controls, and they have values and a schema. So naming should reflect that.

Reply:
but those are not controls that include both, it's a weird hybrid made for stateless triggering, it's a higher level map transmitted in a trigger including a stepid --> controlId --> value nested map, not used anywhere else, I will rename, but the rest of the assumption is not correct, just a weird naming confusing all over